utils/build-galaxy-release.sh

```diff
@@ -107,6 +107,15 @@ sed -i -e "s/name: .*/name: \"$collection\"/" galaxy.yml
 
 find . -name "*~" -exec rm {} \;
 
+
+echo "Creating CHANGELOG.rst..."
+     #shellcheck disable=SC2207
+     REVS=($(git rev-list --tags --max-count=2))
+     #shellcheck disable=SC2207,SC2068
+     TAGS=($(git describe --tags --abbrev=0 ${REVS[@]}))
+     "$(dirname "$0")/changelog" --tag "${TAGS[1]}" --last "${TAGS[0]}" "${TAGS[0]}" > CHANGELOG.rst
```

Comment:
`--last` is used here for the `changelog` script.

Reply:
My bad... originally I implemented with "--last" but it was a worse name than "--upto", so I changed, but forgot to change it also here.

Thank you for noticing this.